classes/blocks/class-splittwocolumns.php

```diff
@@ -78,10 +80,10 @@ public function __construct() {
 			self::BLOCK_NAME,
 			[
 				'editor_script'   => 'planet4-blocks',
-				'attributes'      => static::$attributes,
+				'attributes'      => self::ATTRIBUTES,
```

Comment:
If you use `self` then this is treated as the string literal of the class name. `static` is only useful on a parent class if you want to use the member of a child class, and is evaluated runtime.

Reply:
https://www.php.net/manual/en/language.oop5.late-static-bindings.php